.nvmrc

```diff
@@ -1 +1 @@
-16.13
\ No newline at end of file
+18.15
```

Comment:
Node v16 is in maintenance mode since Oct, 2022; and v18 is the current active LTS (and it is also the first version with `findLastIndex` included 🙂)